docs/iris/src/userguide/cube_maths.rst

```diff
@@ -1,6 +1,6 @@
-======================
-Basic cube mathematics
-======================
+================
+Basic cube maths
+================
 
 
 The section :doc:`navigating_a_cube` highlighted that 
```

Comment:
I started reading this section and hadn't realised I was in a specific example, and I started reading the previous section for the start of the example, and then found myself second guessing the example
Given that this epitomises the value of Iris for tieing metadata to data, and having the ability to do science without having to concern oneself with it, I think it is worth improving this introduction.

```suggestion
 The section Navigating a cube highlighted that every cube has a data attribute; this attribute can then be manipulated directly.  However, the problem with manipulating the data directly is that other metadata may become inconsistent.  For example, assume we have the following temperature data and we wish to convert it to units of Centigrade::
  
   >>> print(cube.data)
   293.15
   >>> print(cube.units)
   “kelvin”

We could manipulate the data directly as follows, but the units would remain in “kelvin” unless we rectify the units attribute too::

   cube.data -= 273.15
   cube.units = “celsius”

In order to reduce the likely hood of metadata becoming inconsistent, we can use the :meth:`iris.cube.Cube.convert_units` method to automatically convert a cube’s data and update its units attribute. So, the two steps above can be achieved by::

   cube.convert_units('celsius')

This issue is quickly exacerbated when carrying out more complex, fundamental arithmetic operations such as addition, subtraction, division and multiplication.  However, applying the functions directly to the cube, using “cube maths”, alleviates the problem just as applying the iris.cube.Cube.convert_units()  does in the example above.

This issue is quickly exacerbated when carrying out more complex, fundamental arithmetic operations such as addition, subtraction, division and multiplication.  However, applying the functions directly to the cube, using “cube maths”, alleviates the problem just as applying the iris.cube.Cube.convert_units()  does in the example above.
```

Reply:
"likelihood" should be one word, and the last paragraph is duplicated here.

Reply:
@abooton I really didn't want to start straying into other parts of the docs and rewriting them... mainly because this PR is getting rather overwhelming. Perhaps raise an issue to tackle this separately? Or even better a PR of your own? :stuck_out_tongue_winking_eye: